examples/Demo_Eigendistortion.ipynb

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
<p>why does <code>k&gt;1</code>  converge faster than <code>k=1</code>?  that's unintuitive for me</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LabForComputationalVision/plenoptic/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>When k&gt;1, each step has an additional 'orthogonalize against all other vectors' step. This carries some extra information about where eigenvectors should settle to and generally helps speed things up.</p><p> </p>

Reply:
<p>I feel like we should discuss that somewhere. Doesn't need to be here, but should be documented and linked to from here.</p>

Reply:
And if you don't want to insert it here or create the docs now, maybe create an issue for the eigendistortion documentation and add this point to it?

Reply:
This is now an issue #89.